sympy/polys/ring_series.py

```diff
@@ -1,19 +1,15 @@
-from sympy.polys.domains import QQ, EX, ExpressionDomain
+from sympy.polys.domains import QQ, EX
 from sympy.polys.rings import PolyElement, ring
 from sympy.polys.polyerrors import DomainError
 from sympy.polys.monomials import (monomial_min, monomial_mul, monomial_div,
-    monomial_ldiv)
+                                   monomial_ldiv)
```

Comment:
4 spaces of indent are generally okay for us, so this is not necessary.
In face we have a preference of 4 spaces over hanging indent, so this would be marginally worse.